akka-http-caching/src/main/scala/akka/http/javadsl/server/directives/CachingDirectives.scala

```diff
@@ -70,5 +70,3 @@ class CachingDirectives {
   def routeCache[K](settings: CachingSettings): Cache[K, RouteResult] =
     JavaMapping.toJava(D.routeCache[K](settings))
 }
-
-object CachingDirectives extends CachingDirectives
```

Comment:
Didn't find a way for the Java DSL to expose the directives as both an extendable class and static methods.